### ISSUE TITLE
Dashboard doesn't work with GitLab 11

### ISSUE THREAD

**Issue by iskren-dimov** [ci-dashboard/gitlab-ci-dashboard#9]
We have upgraded our GitLab to 11.0.2-ee and the dashboard stopped working.
Do you have any plans to add support for version 11 anytime soon?

**Comment by stefan-it** [comment]
You should definitely use API route in version 4. GitLab 11 removed `v3` route (see [here](https://gitlab.com/gitlab-org/gitlab-ce/issues/36819)).

**Comment by iskren-dimov** [comment]
Thanks for pointing that out.  I have updated the configuration, so we are now using V4 of the API. 2 of the projects are loaded on the dashboard, but then an error is shown:

"Something went wrong. Make sure the configuration is ok and your Gitlab is up and running. Details: Cannot read property '0' of null"

I guess not all endpoints of the V4 API are properly implemented.

**Comment by carlsim0n** [comment]
> "Something went wrong. Make sure the configuration is ok and your Gitlab is up and running. Details: Cannot read property '0' of null"

I've just created a pull request https://github.com/emilianoeloi/gitlab-ci-dashboard/pull/10 fixing an issue that caused this problem when a repository has one or more tags. Maybe one of your repos has multiple tags? 

Hopefully we can get this fix into the master branch asap.

**Comment by chillinski** [comment]
I use version 6.4.4 of gitlab-ci-dashboard with version 11.7.5 of GitLAB (v4 API) and the following message is displayed on the board:
```
Something went wrong. Make sure the configuration is ok and your Gitlab is up and running.
Details: s(...).exec(...) is null 
```

**Comment by marek-ganko** [comment]
In the meantime until my fix in PR #16 will be merged, you can use temporary published package with fixed for GitlabCI API v4: https://www.npmjs.com/package/gitlab-ci-v4-dashboard
